### ISSUE TITLE
dependency "got" 11.8.5 has moderate severity vulnerability

### ISSUE THREAD

**Issue by schmulschubiak** [BePo65/license-report#91]
Hej,
dependency "got" in version 11.8.5 has moderate severity vulnerability:

```
got  <11.8.5
Severity: moderate
Got allows a redirect to a UNIX socket - https://github.com/advisories/GHSA-pfrx-2q88-qq97
fix available via `npm audit fix`
node_modules/got

1 moderate severity vulnerability
```

thanks! Markus :)

**Comment by BePo65** [comment]
I do know :-( The problem is that got v12 is n esm only release. To switch, we would have to drop support for node v12, which is out of maintenance state since April 30th 2022. The question is when should we do that and I agree that this should be done asap.

I am trying to remove the dependency on the lock file (issue #84); when this topic is finished the switch over to got v12 will be the next thing.

**Comment by BePo65** [comment]
fixed by pr #96
